cookbook/select-values-provider.rst

```diff
@@ -0,0 +1,85 @@
+Create a select values provider
+===============================
+
+An example :doc:`../reference/content-types/single_select` or :doc:`../reference/content-types/select` values provider could look like the following:
+
+.. code-block:: php
+
+    <?php
+
+    namespace App\Content\Select;
+
+    class EventTypeSelect {
+        /**
+         * @return array<int, array{name: string, title: string}>
+         */
+        public function getValues(): array
+        {
+            return [
+                [
+                    'name' => 'concert',
+                    'title' => 'Concert',
```

Comment:
can we return a localized title?

Reply:
Not sure I never tried it and it seems the expression language does currently not support getting the `locale`.